README.md

```diff
@@ -10,7 +10,7 @@ Foundry and Kubernetes (K8S) environment.
 
 ## Requirements
 
-- Java `≥ 8`
+- Java `≥ 17`
```

Comment:
I would still want to have a note somewhere which I can show to the poor Java 8 users.

Something like...?

```suggestion
- Java `≥ 8` (until `0.10.5`)
- Java `≥ 17` (with `0.10.6` and later)
```